modules/distributor/distributor.go

```diff
@@ -87,6 +89,14 @@ var (
 	}, []string{discardReasonLabel, "tenant"})
 )
 
+// rebatchedTrace is used to more cleanly pass the set of data
+type rebatchedTrace struct {
+	id    []byte
```

Comment:
Is `id` used?  I see `trace.id` used, but maybe missed where `id` is used.

Reply:
Yup, it's used on line 333 where the distributor creates the batches it sends to the ingesters.